app.py

```diff
@@ -0,0 +1,20 @@
+from model import DummyModel
+from flask import Flask, request, Response
+import json
+
+
+model = DummyModel()
+app = Flask(__name__)
+
+
+@app.route('/', methods=['GET', 'POST'])
+def process():
+    if request.method == 'GET':
+        return Response(response=json.dumps({"response": "ok"}), status=200, mimetype="application/json")
+    else:
+        texts = request.json["texts"]
+        response = model.process(texts)
+        return Response(response=json.dumps({"response": response}), status=200, mimetype="application/json")
+
```

Comment:
`jsonify` should do the same: https://flask.palletsprojects.com/en/2.2.x/api/#flask.json.jsonify